pkg/idtools/idtools_unix_test.go

```diff
@@ -251,3 +264,120 @@ dockremap:231072:65536`
 		t.Fatalf("wanted 65536, got %d instead", ranges[0].Length)
 	}
 }
+
+func TestGetRootUIDGID(t *testing.T) {
+	uidMap := []IDMap{
+		{
+			ContainerID: 0,
+			HostID:      os.Getuid(),
+			Size:        1,
+		},
+	}
+	gidMap := []IDMap{
+		{
+			ContainerID: 0,
+			HostID:      os.Getgid(),
+			Size:        1,
+		},
+	}
+
+	uid, gid, err := GetRootUIDGID(uidMap, gidMap)
+	assert.NoError(t, err)
+	assert.Equal(t, uid, os.Getegid())
+	assert.Equal(t, gid, os.Getegid())
+
+	uidMapError := []IDMap{
+		{
+			ContainerID: 1,
+			HostID:      os.Getuid(),
+			Size:        1,
+		},
+	}
+	_, _, err = GetRootUIDGID(uidMapError, gidMap)
+	assert.NotNil(t, err, "Expected error, got nil")
+}
+
+func TestToContainer(t *testing.T) {
+	uidMap := []IDMap{
+		{
+			ContainerID: 0,
+			HostID:      os.Getuid(),
```

Comment:
Maybe use a different number since `0` is likely to be the normal uid?

I don't think we need to use `os.Getuid()` here, right? We could use any number as long as it's passed to `toContainer()` later? Maybe just use a hardcoded number?

Reply:
Yes; I was thinking the same.. mapping 0 to 0 is not that interesting for the `root` case; I think a hardcoded const for a test UID/GID is a better option